blogposts/2021/how-to-not-break-a-search-engine-or-what-i-learned-about-unglamorous-engineering.md

```diff
@@ -1,20 +1,25 @@
 ---
-title: "How Not to Break a Search Engine or: What I Learned About Unglamorous Engineering"
+title: "How not to break a search engine or: What I learned about unglamorous engineering"
+description: "When we switched to a new search query parser in September 2020, you'd never know that
```

Comment:
Took a stab at this so we have a custom summary for the blog homepage and when the post is shared on social. Feel free to edit